lib/coinbase/address.rb

```diff
@@ -85,6 +85,9 @@ def transfer(amount, asset_id, destination)
 
       transfer = create_transfer(amount, asset_id, destination_address)
 
+      # Return here if ServerSigner is expected to sign the transfer.
```

Comment:
Thoughts on:

If a server signer is managing keys, it will sign and broadcast the underlying transfer transaction out of band.